src/statmech.jl

```diff
@@ -142,14 +144,15 @@ function molecular_equilibrium(MEQs, T, nₜ, nₑ; x0=nothing)
     end
 
     #numerically solve for equlibrium.
-    sol = nlsolve(MEQs.equations(T, nₜ, nₑ), x0; iterations=200, store_trace=true, ftol=nₜ*1e-12,
+    sol = nlsolve(MEQs.equations(T, nₜ, nₑ), x0; iterations=1_000, store_trace=true, ftol=nₜ*1e-12,
                   autodiff=:forward)
     if !sol.f_converged
-        error("Molecular equlibrium unconverged", sol, "\n", sol.trace)
+        error("Molecular equlibrium unconverged. (`", sol)
```

Comment:
```suggestion
        error("Molecular equlibrium unconverged. (`", sol, "`)")
```
Minor formatting suggestion

Reply:
I don't think that backtick even does anything.  I've pushed a slightly different formatting.